pkgs/tools/system/dc3dd/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchzip }:
+
+stdenv.mkDerivation rec {
+  pname = "dc3dd";
+  version = "7.3.0";
+
+  src = fetchzip {
+    url = "mirror://sourceforge/dc3dd/dc3dd-${version}.zip";
+    sha256 = "pYr9A9444yC0nYDOf9oDGqgzgQ5Qgtt+J4M7J35HpQg=";
```

SRI hashes are preferred nowadays.

```suggestion
    hash = "...";
```

if `fetchzip` has support for that. I guess it does.